README.md

```diff
@@ -164,8 +164,8 @@ This repository contains many companion libraries to write integration tests wit
 | Fully configurable via configuration file                                                 | ✔️       | ✔️                                                           | ✖️                                                                |
 | Pending tests                                                                             | ✔️       | ✔️                                                           | ✖️                                                                |
 | Iterative testing to diagnose flakiness                                                   | ✔️       | ✖️                                                           | ?                                                                |
-| Test timeout                                                                              | ✔️       | ✖️                                                           | ?                                                                |
-| Test timeout by default                                                                   | ✔️       | ✖️                                                           | ?                                                                |
+| Test timeout                                                                              | ✔️       | C                                                           | ?                                                                |
+| Test timeout by default                                                                   | ✔️       | C                                                           | ?                                                                |
```

Comment:
```haskell
hook = around_ $ maybe (fail "exceeded timeout") pure <=< timeout 1e6
```

Reply:
Same here.
"Test timeout" being `C` is right, but "... by default" can't be `C`.